### PR TITLE
change system time feature

### PR DESCRIPTION
**Pull request by alexgithublab**
This allows to change time easily directly from the GUI.

### CODE REVIEW

**Comment by tlaurion** [comment]
Please chmod +x and amend commit @alexgithublab : not functionally testable.

**Comment by tlaurion** [comment]
@alexgithublab : Also missing notes stating this needs to be UTC/GMT timezone (system clock ) and maybe an online pure HTML5 reference for users to define this without setting local timezone (from their phone...)

See https://github.com/linuxboot/heads/issues/910#issuecomment-734344954

File: initrd/bin/gui-init [hunk 1]
```diff
@@ -438,6 +438,7 @@ show_options_menu()
     'C' ' Reencrypt LUKS container -->' \
     'P' ' Change LUKS Disk Recovery Key passphrase ->' \
     'R' ' Check/Update file hashes on root disk -->' \
+    'h' ' Change system time' \
```
Comment (tlaurion) [inline initrd/bin/gui-init:441]: @alexgithublab 

- Why under gui-init option menu?
  - if under Options menu? Why so low in this menu?
- Why not proposed under TOTP mismatch prompt as replacement for current workflow?

File: initrd/bin/change-time [hunk 2]
```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+#change time using hwclock and date -s
+
+clear
+
+echo -e -n "Please enter the date and time you wish to set\n"
```
Comment (tlaurion) [inline initrd/bin/change-time:6]: GMT/UTC and reference to pure HMTL5 url for user to look for GMT timezone clock with seconds like https://time.is/GMT or better

**Comment by alexgithublab** [comment]
@tlaurion Thank you for advice, this should be better now.

**Comment by tlaurion** [comment]
I'm fine with this @JonathonHall-Purism ?

**Comment by tlaurion** [comment]
Will test https://github.com/linuxboot/heads/pull/1719#issuecomment-2250411862 against his PR which needs to be rebased on master

**Comment by tlaurion** [comment]
> Will test [#1719 (comment)](https://github.com/linuxboot/heads/pull/1719#issuecomment-2250411862) against his PR which needs to be rebased on master

Cannot apply because branch is master. Creating superseeding PR with co-authoring.
@alexgithublab please read and comment https://github.com/linuxboot/heads/blob/master/CONTRIBUTING.md

**Comment by tlaurion** [comment]
Replaced by https://github.com/linuxboot/heads/pull/1737

**Pull request closed without merging**